contrib/templates/scala-job/template/{{.project_name}}/resources/.gitkeep

```diff
@@ -1,3 +0,0 @@
-
-This folder is reserved for Databricks Asset Bundles resource definitions.
-
```

Comment:
This file should stay here, by convention it makes sure that there's always a "resources" dir even if a bundle doesn't have any resources.

Reply:
Fixed.